data/json/items/armor/arms_armor.json

```
@@ -860,12 +884,36 @@
     "material": [ "lc_steel_chain" ],
     "color": "light_red",
     "flags": [ "VARSIZE", "STURDY", "OUTER" ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "volume_encumber_modifier": 0,
+        "ablative": true,
+        "max_contains_volume": "2200 ml",
+        "max_contains_weight": "20 kg",
+        "max_item_length": "65 cm",
+        "moves": 1500,
+        "description": "You can wear some armor over chainmail with out penalty",
```

```suggestion
        "description": "You can wear some armor over chainmail without penalty",
```

This goes for all the descriptions in this file it would appear.